pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -1533,12 +1562,13 @@ func TestNameAndSelectorDaemonSetLaunchesPods(t *testing.T) {
 
 // DaemonSet with node selector that matches some nodes, and node name that matches a different node, should do nothing.
 func TestInconsistentNameSelectorDaemonSetDoesNothing(t *testing.T) {
+	_, ctx := ktesting.NewTestContext(t)
```

Comment:
still missing the context move here (https://github.com/kubernetes/kubernetes/pull/113622#discussion_r1080692221)